book/src/SUMMARY.md

```diff
@@ -15,6 +15,7 @@
     - [F*]()
     - [Coq]()
     - [`libcore`]()
+- [AST](contributing/ast_ebnf.md)
```

Comment:
Shouldn't this be moved in the section contributing? Or be named something like "Input language"?

For now, I'd say we should move it to Contributing